teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(age, name, parent_permission, specialization)
+    super(age, name, parent_permission)
+    @specialization = specialization
+  end
```

Comment:
- [ ] As om the last method, please kindly make the default name "unknown" and parent_permission default to "true".

Like this:

```ruby
  def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
```